### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by JamesJeffryes**
2 Changes:
- Bumps the version and adds comments to the Release Notes
- Makes sure the scratch folder itself is not removed during cleaning

### CODE REVIEW

File: RELEASE_NOTES.txt [hunk 1]
```diff
@@ -27,7 +38,7 @@ NEW FEATURES:
 - Updates to the testing environment
 - "kb-sdk init" now generates a RELEASE_NOTES.md file in new module
 
-VERSION: 0.1.1 through 0.1.16 ( 11/19-2015 through 5/17/2017)
+VERSION: 1.0.1 through 1.0.16 ( 11/19-2015 through 5/17/2017)
```
Comment (jayrbolton) [inline RELEASE_NOTES.txt]: Was this just a mistake/typo people made in previous updates?

File: src/java/us/kbase/templates/module_makefile.vm.properties [hunk 2]
```diff
@@ -122,7 +122,7 @@ build-test-script:
 	echo 'export KB_DEPLOYMENT_CONFIG=$$script_dir/../deploy.cfg' >> $(TEST_DIR)/$(TEST_SCRIPT_NAME)
 	echo 'export KB_AUTH_TOKEN=`cat /kb/module/work/token`' >> $(TEST_DIR)/$(TEST_SCRIPT_NAME)
 	echo 'echo "Removing temp files..."' >> $(TEST_DIR)/$(TEST_SCRIPT_NAME)
-	echo 'rm -rf $(WORK_DIR)' >> $(TEST_DIR)/$(TEST_SCRIPT_NAME)
+	echo 'rm -rf $(WORK_DIR)/*' >> $(TEST_DIR)/$(TEST_SCRIPT_NAME)
```
Comment (jayrbolton) [inline src/java/us/kbase/templates/module_makefile.vm.properties]: good call

**Comment by jayrbolton** [comment]
I just had one question, otherwise looks good to me

File: RELEASE_NOTES.txt (hunk 1 above)
Comment (MrCreosote) [reply RELEASE_NOTES.txt]: No, https://semver.org/#spec-item-4

Comment (MrCreosote) [reply RELEASE_NOTES.txt]: this should not be changed as the numbers are built into the codebase

**Comment by JamesJeffryes** [comment]
@jayrbolton Responded. Thanks!

File: RELEASE_NOTES.txt (hunk 1 above)
Comment (MrCreosote) [reply RELEASE_NOTES.txt]: fair enough then

Comment (jayrbolton) [reply RELEASE_NOTES.txt]: That's what I figured.

**Pull request merged**